lightning_hpo/commands/experiment/run.py

```diff
@@ -24,6 +24,12 @@ def run(self) -> None:
             default=[],
             help="List of requirements separated by a comma or requirements.txt filepath.",
         )
+        parser.add_argument(
+            "--packages",
```

Comment:
Should we expose the build commands instead to be more generic ?

Reply:
Yeah, I went back and forth on this. Let's see how far we get with just packages and we'll eventually extend.